templates/partial/_standard-navigation.html

```diff
@@ -0,0 +1,74 @@
+<header id="standard-navigation" class="p-navigation is-dark">
+  <div class="row">
+    <div class="p-navigation__banner col-3 u-no-padding--left">
+      <div class="p-navigation__tagged-logo">
+        <a class="p-navigation__link" href="/">
+          <div class="p-navigation__logo-tag">
+            <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="Canonical" width="95">
+          </div>
+          <span class="p-navigation__logo-title">Canonical</span>
+        </a>
+      </div>
+      <a href="#standard-navigation" class="p-navigation__toggle--open p-global-nav-titles" title="menu">Menu</a>
+      <a href="#standard-navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
+    </div>
+
+    <nav class="p-navigation__nav col-9" aria-label="main-navigation">
+      <ul class="p-navigation__items js-show-nav" style="margin-left: -.75rem;">
+        <li class="p-navigation__item--dropdown-toggle" id="company-nav">
+          <a href="/company" aria-controls="company-menu" class="p-navigation__link">Company</a>
+          <ul class="p-navigation__dropdown" id="company-menu" aria-hidden="true">
```

Comment:
why should these links be hidden from screen readers?

Reply:
Not entirely sure, I just moved [existing code](https://github.com/canonical/canonical.com/blob/main/templates/partial/_navigation.html#L43) I'll need to investigate.

Reply:
Ok, I see why, because this element is hidden. It will change to "false" once the company link is clicked, the dropdown will show.

Reply:
okay, good that you checked.